regress/pesterTests/SFTP.Tests.ps1

```diff
@@ -222,7 +222,7 @@ Describe "SFTP Testcases" -Tags "CI" {
            Set-Content $batchFilePath  -Encoding UTF8 -value $commands
            $str = $ExecutionContext.InvokeCommand.ExpandString(".\sftp $($Options) $($LogonStr) > $outputFilePath")
            $client.RunCmd($str)
-           $outputFilePath | Should Contain ([RegEx]::Escape((join-path $tmpDirectoryPath1 $tmpFileName1).replace("\", "/")))
+           Test-Path ([RegEx]::Escape((join-path $tmpDirectoryPath1 $tmpFileName1).replace("\", "/")))
```

Comment:
same as above. this line does not validate the file exist. if you want to validate the file exist, you probably need to (Test-Path ...) | Should be $true.